### PR TITLE
feat!: change from autoware_auto_msgs to autoware_msgs

### PR DESCRIPTION
**Pull request by yukkysaito**
## Description

following [change from autoware_auto_msgs to autoware_msgs](https://github.com/orgs/autowarefoundation/discussions/3862)

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [ ] I've confirmed the [contribution guidelines].
- [ ] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].
- [ ] The PR has been properly tested.
- [ ] The PR has been reviewed by the code owners.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by yukkysaito** [comment]
@cyn-liu @mitsudome-r could you resolve conflicts?
https://github.com/orgs/autowarefoundation/discussions/3862#discussioncomment-9626700

**Comment by veqcc** [comment]
Could you split the Pull Request?

**Review by soblin — APPROVED** [review]

**Comment by yukkysaito** [comment]
> Could you split the Pull Request? Thank you.

@veqcc Thank you for the suggestion. Given the nature of this PR, it is almost impossible to separate it by components. Since it involves changing the interfaces between the base components, splitting it into PRs that each function for one component would ultimately require changing the whole system. Therefore, we have allocated a long period and prepared a transition period for this purpose.
This discussion has been going on for over six months and today is the end of the transition period. If there was a proposal, we wanted it a little sooner.
Reference : https://github.com/orgs/autowarefoundation/discussions/3862#discussion-5671332

**Comment by veqcc** [comment]
@yukkysaito 
> Thank you for the suggestion. Given the nature of this PR, it is almost impossible to separate it by components. Since it involves changing the interfaces between the base components, splitting it into PRs that each function for one component would ultimately require changing the whole system. Therefore, we have allocated a long period and prepared a transition period for this purpose.
This discussion has been going on for over six months and today is the end of the transition period. If there was a proposal, we wanted it a little sooner.

Thank you for your reply.
I understand the nature and your long efforts.

Then, my another request is that please at least make it clear which components or interfaces each reviewer should review.
For someone like me, it is unclear which module to ckeck, and I cannot approve forever.

**Comment by YossyTaka** [comment]
I tend to agree with @veqcc on this one.
Are all changes find-and-replace type of changes? If yes, I would be willing to let this one thru. If the answer is no, then I would reconsider how you approach this type of PRs in general. 

At the very least, please add the description of how and how long you tested the changes in a private branch prior to merging it to the mainline. I believe that's the least you can do to be a role model for more junior engineers :)

**Comment by yukkysaito** [comment]
@YossyTaka 
Basically, these should be replacement changes with no changes to the algorithm.

Of course, that's our intention. Since this will be an individual PR, we plan to compile the tests here. The original discussion already mentions that we will conduct operational checks for the planning simulator (tutorial) and logging simulator (tutorial), but if you have any suggestions, please let us know.
https://github.com/orgs/autowarefoundation/discussions/3862

File: build_depends.repos
Comment (yukkysaito) [inline build_depends.repos:6]:
```suggestion
    version: main
```

File: build_depends.repos
Comment (yukkysaito) [inline build_depends.repos:31]:
```suggestion
    version: tier4/universe
```

File: build_depends.repos
Comment (yukkysaito) [inline build_depends.repos:14]:
```suggestion
    version: main
```

**Comment by yukkysaito** [comment]
Relevant components were split up and reviewed component by component by AWF maintainers.
@mitsudome-r Can you share evaluation result?

**Comment by mitsudome-r** [comment]
NOTE: We have confirmed that all the test results in Web.Auto Evaluator were same between the main branch and this branch before merging.

Before (442/617 passing): https://evaluation.tier4.jp/evaluation/reports/95a837bd-c785-55d6-90e7-1eeed9376ed4?project_id=awf
After (443/617 passing): https://evaluation.tier4.jp/evaluation/reports/4187dcf0-f1a6-561a-9e1d-98c361cce3d0?project_id=awf

There is a small difference in the result, but this is likely to be the fact that autoware and simulator not being deterministic.

**Comment by yukkysaito** [comment]
This PR will be closed

**Pull request closed without merging**
